src/middleware/mod.rs

```diff
@@ -341,6 +341,18 @@ impl JsonSchema for TypedValue {
             ..Default::default()
         };
 
+        let pod_id_schema = schemars::schema::SchemaObject {
```

Comment:
This is unrelated but I noticed that this is missing during testing. The JSON schema is used for validation and TypeScript code generation in the Podnet client, and I noticed that it wasn't rendering PodIds correctly because this is missing.